### PR TITLE
Hunting Queries and Parser updated from the Monitoring SQL blog Post

### PR DESCRIPTION
**Pull request by Iftekharh**
Fixes #

## Proposed Changes
https://techcommunity.microsoft.com/t5/azure-sentinel/monitoring-sql-server-with-azure-sentinel/ba-p/1502960
  -
  -
  -

### CODE REVIEW

**Comment by Iftekharh** [comment]
Thanks Shain, updated hunting queries as per Query Style Guide and validated YAML syntax

**Comment by Iftekharh** [comment]
Updated hunting queries as per feedback.

**Comment by Iftekharh** [comment]
Hi Shain, thank you for valuable feedback, updated the queries as per guidelines.

File: Hunting Queries/SQLServer/SQL-MultipleFailedLogon_InShortSpan.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  - CredentialAccess
+relevantTechniques:
+  - T1110
+query: |
+
+        // SQLEvent is not the table name, it is the function name that should already be imported into your workspace.
+        // The underlying table where the data exists is the Event table.
+        // the timeframe and threshold can be changed below as per requirement
+        //
+        SQLEvent
```
Comment (shainw) [inline Hunting Queries/SQLServer/SQL-MultipleFailedLogon_InShortSpan.yaml:22]: Missed adding in - let TimeFrame = 1d;

File: Hunting Queries/SQLServer/SQL-New_UserCreated.yaml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+query: |
+
+        // This detection query is based on the SQLEvent KQL Parser provided at https://github.com/Azure/Azure-Sentinel/tree/master/Parsers/SQLSever
+        // Detailed blog post on Monitoring SQL Server with Azure Sentinel https://techcommunity.microsoft.com/t5/azure-sentinel/monitoring-sql-server-with-azure-sentinel/ba-p/1502960
+        // This query checks for new user account created on SQL Server using the SQLEvent() parser
+        //
+        SQLEvent
+        | where Statement has "Create Login"
+        | parse Statement with "CREATE LOGIN [" TargetUser:string
+                              "]" *
```
[the remark this answers is not in the document]
Comment (shainw) [reply Hunting Queries/SQLServer/SQL-New_UserCreated.yaml:26]: Still need to fix this.

[the remark this answers is not in the document]
Comment (Iftekharh) [reply Hunting Queries/SQLServer/SQL-New_UserCreated.yaml]: Corrected

File: Hunting Queries/SQLServer/SQL-MultipleFailedLogon_InShortSpan.yaml (hunk 1 above)
Comment (Iftekharh) [reply Hunting Queries/SQLServer/SQL-MultipleFailedLogon_InShortSpan.yaml:24]: Query Updated

**Pull request merged**